.travis.yml

```diff
@@ -24,10 +24,13 @@ matrix:
       python: 3.7
     - env: TOXENV=py38
       python: 3.8
+      dist: bionic
     - env: TOXENV=extra-deps
       python: 3.8
+      dist: bionic
     - env: TOXENV=py38-asyncio
       python: 3.8
+      dist: bionic
```

Comment:
heh, the latest is already 20.04 :)

Reply:
Yeah, tell [*them*](https://docs.travis-ci.com/user/reference/linux/) that :sweat: